### ISSUE TITLE
replace default Edison run

### ISSUE THREAD

**Issue by xylar** [MPAS-Dev/MPAS-Analysis#275]
It seem like the file for the "default" run we use for testing on Edison are disappearing, presumably as they get cleaned up to make room on scratch:
```
> ls mpaso.hist.am.timeSeriesStatsMonthly.00*
mpaso.hist.am.timeSeriesStatsMonthly.0001-01-01.nc
mpaso.hist.am.timeSeriesStatsMonthly.0001-02-01.nc
mpaso.hist.am.timeSeriesStatsMonthly.0001-06-01.nc
mpaso.hist.am.timeSeriesStatsMonthly.0001-07-01.nc
mpaso.hist.am.timeSeriesStatsMonthly.0001-08-01.nc
mpaso.hist.am.timeSeriesStatsMonthly.0001-10-01.nc
mpaso.hist.am.timeSeriesStatsMonthly.0002-06-01.nc
mpaso.hist.am.timeSeriesStatsMonthly.0002-08-01.nc
mpaso.hist.am.timeSeriesStatsMonthly.0002-11-01.nc
mpaso.hist.am.timeSeriesStatsMonthly.0003-02-01.nc
mpaso.hist.am.timeSeriesStatsMonthly.0003-04-01.nc
mpaso.hist.am.timeSeriesStatsMonthly.0003-07-01.nc
mpaso.hist.am.timeSeriesStatsMonthly.0003-08-01.nc
mpaso.hist.am.timeSeriesStatsMonthly.0003-12-01.nc
mpaso.hist.am.timeSeriesStatsMonthly.0004-03-01.nc
mpaso.hist.am.timeSeriesStatsMonthly.0004-05-01.nc
mpaso.hist.am.timeSeriesStatsMonthly.0004-06-01.nc
mpaso.hist.am.timeSeriesStatsMonthly.0004-07-01.nc
mpaso.hist.am.timeSeriesStatsMonthly.0004-12-01.nc
mpaso.hist.am.timeSeriesStatsMonthly.0005-06-01.nc
mpaso.hist.am.timeSeriesStatsMonthly.0005-07-01.nc
mpaso.hist.am.timeSeriesStatsMonthly.0005-08-01.nc
mpaso.hist.am.timeSeriesStatsMonthly.0005-09-01.nc
mpaso.hist.am.timeSeriesStatsMonthly.0005-10-01.nc
mpaso.hist.am.timeSeriesStatsMonthly.0005-11-01.nc
mpaso.hist.am.timeSeriesStatsMonthly.0005-12-01.nc
```
We should find a good beta2 run to use instead.  @milenaveneziani, do you know of one?

**Comment by xylar** [comment]
Great, trying that one now.

**Comment by xylar** [comment]
@milenaveneziani, this beta3 run has the MOC analysis member on, but we don't support that yet in MPAS-Analysis.  How would you like to handle this?  Should we ignore the online MOC for now and update to pay attention to the online MOC once we have the appropriate code?

**Comment by milenaveneziani** [comment]
Were the 3d velocities needed to compute post-processed MOC saved?

Anyway, I think I should add the MOC online script sooner rather than later. I will talk with @vanroekel about it after the break (I will be off tomorrow).

**Comment by xylar** [comment]
I have no idea but I'm trying it now.  We'll find out soon.

**Comment by xylar** [comment]
Yep, my test finished successfully so the velocities must still be included in the monthly mean output.

**Comment by milenaveneziani** [comment]
@xylar, I am looking at this beta2 run:
`/global/cscratch1/sd/golaz/ACME_simulations/20170915.beta2.A_WCYCL1850S.ne30_oECv3_ICG.edison`
and 153 years of mpaso files are still there, in the short-term archive.
Which beta2 run were you using for testing?

**Comment by xylar** [comment]
Should be addressed by #276
